applications/crossbar/src/crossbar_maintenance.erl

```diff
@@ -271,6 +272,20 @@ find_account_by_realm(Realm) ->
 %%
 %% @end
 %%--------------------------------------------------------------------
+-spec find_account_by_id(input_term()) ->
+                                   {'ok', ne_binary()} |
+                                   {'error', term()}.
+find_account_by_id(Id) when not is_binary(Id) ->
+    find_account_by_id(wh_util:to_binary(Id));
+find_account_by_id(Id) ->
+    print_account_info(wh_util:format_account_id(Id, 'encoded')).
```

Comment:
I would do the opposite and have a guard like `when is_binary…` then an unguarded clause